melusine/models/train.py

```diff
@@ -361,12 +362,54 @@ def predict_proba(self, X, **kwargs):
         Parameters
         ----------
         X : pd.DataFrame
-
+        confidence_interval : float, optional
+            between [0,1], the confidence level of the interval.
+            Only available with tensorflow-probability models.
         Returns
         -------
-        np.array
+        score : np.array
```

Comment:
If the user set **confidence_interval** option then the function return additional outputs.
I hesitated to make the number of outputs depend on an option but I remind numpy did the same pattern with np.unique(return_counts=True) so I guess it is okay ?

Reply:
I'd say yes, if the user activates the prediction_interval option, he is probably an advanced user and he should be aware that the number of outputs changes